### ISSUE TITLE
Was not able to save the URL using singlefile. The reported error was No such file or directory

### ISSUE THREAD

**Issue by Bristopher** [davekats/canvas-student-data-export#11]
**Describe the bug**\
`% python export.py`
```
Downloading course list page Was not able to save the URL https://xxxx.instructure.com/courses/ using singlefile. The reported error was No such file or directory
```
Issue is only with single-file, the rest of the program will still run till completion with expected output/export.

**To Reproduce**\

1. Manual installation: \
`pip install -r requirements.txt`\
`sudo apt install git`\
`npm i github:gildas-lormeau/SingleFile`

2. Run: `python export.py`
3. output snippet below
```
Welcome to the Canvas Student Data Export Tool

Connecting to canvas

Creating output directory: ./output

Getting list of all courses

  Downloading course list page
Was not able to save the URL https://xxxx.instructure.com/courses/ using singlefile. The reported error was No such file or directory
Working on 2022 Fall C: CSE XXX: Algorithms (2022 Fall - C)
  Getting assignments
```
**Expected behavior**\
Save https://xxxx.instructure.com/courses/ to courses_list.html in ./ouput.

**Environment**
- OS: Ubuntu 22.04.1 LTS\
- Browser: chromium-revision@1.0.0\
- puppeteer@17.7.0\
- single-file@1.0.23\
- Node.js: v18.12.1\
- npm: 8.5.1

**Comment by davekats** [comment]
@dj346 Are you able to help debug this issue? I no longer have access to Canvas so it's difficult for me to debug issues like this

**Comment by davekats** [comment]
@Bristopher I no longer have access to Canvas so it will be challenging for me to debug this issue. If you'd like you can schedule a call with me and we can debug the issue from your computer: https://usemotion.com/meet/davidk/hxmsj33

**Comment by dj346** [comment]
@Bristopher Can you go to the bottom of singlefile.py, uncomment out the section at the bottom, and run the singlefile.py? Post any errors you get. Sorry for the late response.

**Comment by Bristopher** [comment]
@davekats @dj346 We're even now haha, been so busy with this last semester. That initial issue I posted above must've been a quirk because out of all the times I've used this that isn't the main issue, the main issue is this:
`Downloading course list page Failed to launch the browser process! spawn chrome ENOENT`

Somehow it can never correctly find chrome (after searching it seems to be a very common problem, I think it's just a puppeteer issue), I've been trying to just dockerize it at this point so I don't have to deal with 
windows-->linux issues, and its good practice to get better at it, but still same issue. If I use the Dockerfile given from gildas-lormeau/single-file-cli to test it finally works, but now there's a new bigger hurdle. 

Ever since a couple years ago a lot Universities started using 2FA and redirect you after you login, even with the cookies file, I believe it momentarily redirects to 2FA portal to check and then continues to the page, so single-file just captures the page of the 2FA portal. Not sure if there's a way to delay single-file or somehow bypass the redirect check if you have the correct cookies, or something like that.
`https://weblogin.XXX.edu/cas/login?service=https%3A%2F%2FXXX.instructure.com%2Flogin%2Fcas`
